threedi_modelchecker/config.py

```diff
@@ -1400,42 +1360,42 @@ def is_none_or_empty(col):
     AllEqualCheck(error_code=330 + i, column=column, level=CheckLevel.WARNING)
```

Comment:
This check should be dropped: `AllEqualCheck` checks if the values in all records are the same. However, you now constain to 1 record.

Reply:
Dropped in a77f89911c7d5cd14adc31001125d207e93c5355